pkg/genericapiserver/genericapiserver.go

```diff
@@ -224,9 +224,12 @@ type Config struct {
 	// The range of IPs to be assigned to services with type=ClusterIP or greater
 	ServiceClusterIPRange *net.IPNet
 
-	// The IP address for the GenericAPIServer service (must be inside ServiceClusterIPRange
+	// The IP address for the GenericAPIServer service (must be inside ServiceClusterIPRange)
 	ServiceReadWriteIP net.IP
 
+	// Port for the GenericAPIServer service.
```

Comment:
Not generic apiserver-- the specific apiserver. E.g., `kubernetes`, `ubernetes`, `mypluginapiserver`